network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/generic-card.html

```diff
@@ -8,7 +8,7 @@
   </div>
   {% block tags %}{% endblock %}
   <a class="h4-heading d-inline-block my-2" href={{ page.url }}>{{ page.title }}</a>
-  <p class="body-small mb-0">
+  <p class="body-small my-0">
```

Comment:
this also affects every other instance of cards: is this the intended spacing for index pages as well as "related cards" on individual blog pages?

Reply:
Yeah, that's a fix because in `wagtail.scss` there's a `margin-top: 1rem` on all `<p>` that are children of an element with the `.cms` class. Because of this, this component's blog cards  just have the extra margin-top removed and as for the other card instances, they're practically unaffected by this change.